File: src/domain/suspense-account.ts
/**
 * Bank A's transit account for interbank transfers. Locally, sending money to
 * Bank B is booked as an ordinary transfer to this account — see
 * migrations/005_create_interbank_transfers.sql for why.
 */
export const SUSPENSE_ACCOUNT_ID = '00000000-0000-0000-0000-000000000001';

File: bank-b/app/domain/suspense_account.py
"""
Bank B's own transit account. Crediting a payee for money arriving from Bank A
still has to keep Bank B's local double-entry ledger balanced, so the credit
is paired with a debit to this account instead of appearing out of nowhere —
see migrations/003_create_incoming_transfers.sql.
"""

SUSPENSE_ACCOUNT_ID = "00000000-0000-0000-0000-000000000001"
